README.md

```diff
@@ -7,6 +7,8 @@ This is the source repository for the Baïkal CalDAV and CardDAV server.
 
 Head to [sabre.io/baikal][2] for information about installation, upgrading and troubleshooting.
 
+A German tutorial about installation of Baïkal, using it together with Thunderbird, Android and DavX5, sharing calendars, database maintenance and security can be found [here][6]. It's focus is for people with not so much IT experience. Therefore it is very detailed, step by step with a RaspberryPI used as server.
```

Comment:
I think this should be `Its`, not `It's`. `It's` is a short form for `It is`, if I remember correctly.